packages/swingset-runner/demo/exchangeBenchmark/exchanger.js

```diff
@@ -1,10 +1,12 @@
 /* global harden */
```

Comment:
the mode of this file changed.. not a problem but maybe figure out what's changing it